presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the join operator to prevent hanging when spill is enabled and the probe side finishes before the hash builder starts
+* Add new spilling strategy to spill when a combination of user, system and recovable memory on a node exceeds ``max_total_memory_per_node``. This can be enabled by setting the system configuration ``experimental.spiller.task-spilling-strategy`` to ``PER_QUERY_MEMORY_LIMIT``.
+* Add support for logging formatted prepared query. This can be enabled by setting session property ``log_formatted_query_enabled`` to ``true``.
+* Add support for returning partial results for the queries by setting ``partial_results_enabled`` session property. Additionally ``partial_results_max_execution_time_multiplier``, ``partial_results_completion_ratio_threshold``  session properties can be set to configure the max execution time multiplier and minimum completion ratio threshold for the queries.
+* Add automatic query retry functionality for transient failures. This can be enabled by setting ``per-query-retry-limit`` to a non-zero integer to indicate the per query retry count.
+* Add support to coordinator endpoint ``/v1/info/state`` to return ``ACTIVE`` when the coordinator is not shutting down and the cluster has the minimum required workers.
+* Add functions :func:`chisquared_cdf` and :func:`inverse_chisquared_cdf`.
+
+Security Changes
+________________
+
+* Add file based password authenticator plugin. See :doc:`security/password-file`.
+
+Hive Changes
+____________
+* Update Alluxio cache config property ``cache.alluxio.timeout-enabled`` to be ``true`` by default.
+
+JDBC Changes
+____________
+* Add support to serialize ``PrestoArray`` to a string.
+
+Elasticsearch Connector
+_______________________
+* Add support for running base32 encoding ElasticSearch queries that are passed through. See :doc:`connector/elasticsearch` on how to use it.
```

Comment:
highlight worthy

Reply:
@aweisberg - how about this one - 

```
Add support for running base32 encoded ElasticSearch queries. See :doc:`connector/elasticsearch`.
```

Also can you check if we are able to link to a specific section - specifically the passthrough queries section in the elastic search documentation

Reply:
You should be able to link to a specific section. I spent an hour trying to make it work using https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html but I couldn't make it happen. I tried auto-generating labels, setting an explicit target and linking to it (both with _ and :ref:) and none worked. I went back several versions and didn't find any examples that worked.

A small fix, we need to add a / before connector for it to render as a link.
```
Add support for running base32 encoded ElasticSearch queries. See :doc:`/connector/elasticsearch`.
```